train_model.py

```diff
@@ -47,9 +47,7 @@
 parser.add_argument('--dropout_p_decoder', type=float, help='Dropout probability for the decoder', default=0.2)
 parser.add_argument('--teacher_forcing_ratio', type=float, help='Teacher forcing ratio', default=0.2)
 parser.add_argument('--attention', choices=['pre-rnn', 'post-rnn'], default=False)
-parser.add_argument('--attention_method', choices=['dot', 'mlp', 'concat', 'hard'], default=None)
-parser.add_argument('--use_attention_loss', action='store_true')
-parser.add_argument('--scale_attention_loss', type=float, default=1.)
+parser.add_argument('--attention_method', choices=['dot', 'mlp', 'concat'], default=None)
 parser.add_argument('--xent_loss', type=float, default=1.)
```

Comment:
I believe `xent_loss` was introduced to balance the target loss vs. AG loss. Maybe this parameter is obsolete now.